siliconcompiler/core.py

```diff
@@ -4029,6 +4030,7 @@ def run(self):
                     self.set('option', 'steplist', pre_remote_steplist['steplist'])
                 else:
                     self.unset('option', 'steplist')
+                self.unset('option', 'scheduler', 'name')
```

Comment:
Why the need to unset this?

Reply:
Could that be used in `remote_run`?

Reply:
We need to un-set it because the local `Chip` object will pick up the `('option', 'scheduler', 'name')` value from the remote run's manifest, and try to run the screenshot step which happens later on a cluster.

If we unset the value during the remote run, it would get written out as `None` in each task's output manifest, and following tasks wouldn't get the correct value to run the job through the scheduler. It would also be risky to have the 'runner' node modify the output manifests after each task is completed, because NFS can be a bit slow to register new files which another machine created, and re-mounting the drive is the only reliable solution I've found for that so far.

That said, I have one new idea after sleeping on this: if we let the `('option', 'scheduler', 'name')` parameter be set on a per-node basis, the server could specify 'slurm' in only those tasks which are in the remote steplist, and leave the default value untouched.